echopop/tests/test_kriging_biomass_apportionment.py

```diff
@@ -0,0 +1,878 @@
+    # total_weight_sex_all
+    total_weight_sex_all_array = np.repeat( 10 , 16 ).astype( np.float64 ).flatten( )
+    total_weight_sex_all_array = np.concatenate( [ total_weight_sex_all_array , np.repeat( 20 , 8 ).astype( np.float64 ).flatten( ) ] )
+
+    expected_output = pd.DataFrame( {
+        'stratum_num': np.repeat( [ 0 , 1 ] , 8 ).astype( np.int64 ) ,
+        'species_id': np.repeat( 19350 , 16 ).astype( np.int64 ) ,
+        'sex': sex_array ,
+        'length_bin': pd.cut( np.tile( [ 12 , 12 , 19 , 19 ] , 4 ) , 
+                              np.linspace( 9 , 21 , 3 ) ) ,
+        'age_bin': pd.cut( np.tile( [ 1 , 2 ] , 8 ) , 
+                           np.array( [ 0.5 , 1.5 , 2.5 ] ) ) ,
+        'weight_all': [ 3.0 , 0.0 , 0.0 , 7.0 , 2.0 , 0.0 , 0.0 , 8.0 ,
+                        4.0 , 0.0 , 0.0 , 6.0 , 1.0 , 0.0 , 0.0 , 9.0 ] ,
+        'weight_adult': [ 0.0 , 0.0 , 0.0 , 7.0 , 0.0 , 0.0 , 0.0 , 8.0 ,
+                          0.0 , 0.0 , 0.0 , 6.0 , 0.0 , 0.0 , 0.0 , 9.0 ] ,
+        'total_weight_sex_all': np.repeat( 10 , 16 ).astype( np.float64 ).flatten( ) ,
+        'total_weight_sex_adult': [ 7.0 , 7.0 , 7.0 , 7.0 , 8.0 , 8.0 , 8.0 , 8.0 ,
+                                    6.0 , 6.0 , 6.0 , 6.0 , 9.0 , 9.0 , 9.0 , 9.0 ] ,
+        'proportion_weight_sex_all': [ 0.3 , 0.0 , 0.0 , 0.7 , 0.2 , 0.0 , 0.0 , 0.8 ,
+                                       0.4 , 0.0 , 0.0 , 0.6 , 0.1 , 0.0 , 0.0 , 0.9 ] ,
+        'proportion_weight_sex_adult': [ 0.0 , 0.0 , 0.0 , 1.0 , 0.0 , 0.0 , 0.0 , 1.0 ,
+                                         0.0 , 0.0 , 0.0 , 1.0 , 0.0 , 0.0 , 0.0 , 1.0 ]
+        } )
+    expected_output[ 'length_bin' ] = pd.IntervalIndex( expected_output[ 'length_bin' ] )
+    expected_output[ 'length_bin' ] = pd.Categorical( expected_output[ 'length_bin' ] , 
+                                                      categories =  expected_output[ 'length_bin' ].unique( ) , 
+                                                      ordered = True )
+    expected_output[ 'age_bin' ] = pd.IntervalIndex( expected_output[ 'age_bin' ] )
+    expected_output[ 'age_bin' ] = pd.Categorical( expected_output[ 'age_bin' ] , 
+                                                 categories = expected_output[ 'age_bin' ].unique( ) , 
+                                                 ordered=True)
+
+    #----------------------------------
+    ### Run tests: `compute_index_aged_weight_proportions`
+    #----------------------------------
+    ### Process the specimen data 
+    ### Check shape 
+    assert obj_props_wgt_len_age_sex.shape == expected_dimensions
+
+    ### Check datatypes
+    assert np.all( obj_props_wgt_len_age_sex.dtypes == expected_output.dtypes )
+    
+    ### Check data value equality
+    assert expected_output.equals( obj_props_wgt_len_age_sex )
+    
+def test_compte_aged_unaged_proportions( ):
```

Comment:
just fixing a tiny typo
```suggestion
def test_compute_aged_unaged_proportions( ):
```